style.css

```diff
@@ -0,0 +1,619 @@
+* {
+  font-size: 2rem;
+  font-family: "Lato", sans-serif;
+}
+
+.container-fluid {
+  background-image: url("../images/Nutrition-Background.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.socialMediaIcons {
+  display: none;
+}
+
+.navItem {
+  position: fixed;
+  left: -100%;
+  width: 100%;
+  height: 100%;
+  gap: 0;
+  color: rgb(51, 47, 47);
+  font-size: 3.6rem;
+  padding: 4rem;
+  text-align: center;
+  background-color: #f7f3f3;
+}
+
+.navItem.active {
+  left: 0;
+  z-index: 1000;
+}
+
+.navItem li {
+  padding: 2rem;
+  border: 3px solid gainsboro;
+  margin-top: 2rem;
+  margin-left: 2rem;
+}
+
+.navItem.active li:hover {
+  border: 3px solid red;
+}
+
+.navItem.active li a:focus {
+  border: 3px solid green;
+}
+
+.navbar-brand {
+  display: none;
+}
+
+.humgurger {
+  display: block;
+  margin-left: 2.5rem;
+  margin-top: 2rem;
+}
+
+.humgurger .bar {
+  background-color: #272a31;
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px;
+  -webkit-transition: all 0.3s ease-in-out;
+  transition: all 0.3s ease-in-out;
+}
+
+.humgurger.active .bar:nth-child(2) {
+  opacity: 0;
+}
+
+.humgurger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+}
+
+.humgurger.active .bar:nth-child(3) {
+  transform: translateY(-8px) rotate(-45deg);
+}
+
+.introduction {
+  margin-top: 6rem;
+  line-height: 4rem;
+}
+
+footer p {
+  display: none;
+}
+
+.introduction p {
+  margin-left: 5.5rem;
+  font-size: 2.5rem;
+  color: #ec5242;
+  font-family: "Lato", sans-serif;
+  font-weight: 400;
+}
+
+.secondHeader {
+  background-image: url("./images/bg2.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  text-transform: uppercase;
+  font-family: "Cocogoose", sans-serif;
+  font-weight: 800;
+  -webkit-font-smoothing: antialiased;
+  margin-left: 5.5rem;
```

Comment:
[OPTIONAL]
- [ ] On mobile you consider fixing the width of your page and that way your page will be a `100%` responsive